applications/GeoMechanicsApplication/tests/dsettlement/fully_saturated_column_uniform_load/ProjectParameters_stage1.json

```
@@ -0,0 +1,251 @@
+{
+  "problem_data": {
+    "problem_name"                    : "mesh",
+    "start_time"                      : -0.001,
+    "end_time"                        : 0,
+    "echo_level"                      : 1,
+    "parallel_type"                   : "OpenMP",
+    "number_of_threads"               : 1
+  },
+  "solver_settings": {
+    "solver_type"                     : "U_Pw",
+    "model_part_name"                 : "PorousDomain",
+    "domain_size"                     : 2,
+    "model_import_settings": {
+      "input_type"                    : "mdpa",
+      "input_filename"                : "mesh"
+    },
+    "material_import_settings": {
+      "materials_filename"            : "MaterialParameters_linear.json"
+    },
+    "time_stepping": {
+      "time_step"                     : 0.001,
+      "max_delta_time_factor"         : 86400000000
+    },
+    "buffer_size"                     : 2,
+    "echo_level"                      : 1,
+    "clear_storage"                   : false,
+    "compute_reactions"               : true,
+    "reform_dofs_at_each_step"        : false,
+    "nodal_smoothing"                 : false,
+    "block_builder"                   : true,
+    "solution_type"                   : "Quasi-Static",
+    "scheme_type"                     : "Backward_Euler",
+    "reset_displacements"             : true,
+    "strategy_type"                   : "line_search",
+    "convergence_criterion"           : "displacement_criterion",
+    "displacement_relative_tolerance" : 0.001,
+    "displacement_absolute_tolerance" : 1.0E-9,
+    "min_iterations"                  : 6,
+    "max_iterations"                  : 30,
+    "number_cycles"                   : 100,
```

I find this quite a high number, I think even 1 should work right, since this converges quickly anyway (same holds for the other stages)?
```suggestion
    "number_cycles"                   : 1,
```